standup_notes/__main__.py

```diff
@@ -28,8 +30,8 @@ def main():
                                              'Otherwise, copies the specified day\'s notes ', action='store_true')
     parser.add_argument('-e', '--edit', help='Edit stand-up notes', action='store_true')
     parser.add_argument('-d', '--delete', help='Delete stand-up notes from inputted date', action='store', type=str)
+    parser.add_argument('-p', '--post', help='Function to post notes to msteams chat', action='store_true')
```

Comment:
I'm wondering if we want to make edit, copy, post, and delete mutually exclusive arguments as well, since there isn't really a use case for doing more than one at a time. The order in which they get executed is also arbitrary, so it doesn't make a whole lot of sense to the user as to what is happening.

Reply:
Right now you can use the copy flag with the edit flag to edit tomorrow's notes while copying today's todo list. So they aren't mutually excusive. I think we could definitely make them mutually exclusive though if we just made a different flag for the edit argument to copy previous notes and I think it would make sense to do so.

Reply:
Mutually exclusive arguments has been added now.
I removed the -c flag if you wanted to copy previous days notes and replaced it with a separate command. It is now -x or --editcopy. The .bash file has been changed to show new argument.